metadata-ingestion/src/datahub/ingestion/source/sql/sql_generic_profiler.py

```diff
@@ -147,7 +147,10 @@ def is_dataset_eligible_for_profiling(
             if last_profiled
             else None
         )
-        if self.config.profiling.profile_if_updated_since_days is not None:
+        if (
```

Comment:
Should we disable "self.config.profiling.profile_if_updated_since_days" (i.e. set to None) if `store_last_profiling_timestamps` is set to True. 
The defaults for `self.config.profiling.profile_if_updated_since_days` config are very conservative  i.e. 1 day and it will interfere with the profiling state handler, which smartly determines the whether to profile a table.

Reply:
thanks, you are absolutely right, I will address this